WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -308,7 +308,7 @@ private void ProcessStatusChange(StatusChangedEventArgs e)
 				{
 					CoinjoinError.NoCoinsToMix => new() { Message = "Waiting for confirmed funds" },
 					CoinjoinError.UserInSendWorkflow => new() { Message = "Waiting for closed send dialog" },
-					CoinjoinError.AllCoinsPrivate => new() { Message = "Hurray!! Your wallet is private" },
+					CoinjoinError.AllCoinsPrivate => new() { Message = "Hurray! All your coins are private" },
```

Comment:
I guess it was intentional not to mention coins here. How about this:

```suggestion
					CoinjoinError.AllCoinsPrivate => new() { Message = "Hurray! All your balance is private" },
```

Reply:
then it should be `Your funds are private`